packages/relay-service/lib/did/did-resolver.js

```diff
@@ -0,0 +1,28 @@
+import {
+  DockResolver,
+  DIDKeyResolver,
+  MultiResolver,
+  UniversalResolver,
+} from '@docknetwork/sdk/resolver';
+import dock from '@docknetwork/sdk';
+
+// Create a resolver in order to lookup DIDs for verifying
+export const universalResolverUrl = 'https://uniresolver.io';
+
+class WalletSDKResolver extends MultiResolver {
```

Comment:
Will refactor this later, need to have a DID resolver shared with all the wallet-sdk modules. Need to be moved to the `core` package. https://dock-team.atlassian.net/browse/DCKA-1502